### PR TITLE
Update revenue_tracking.md for React Native

### PR DESCRIPTION
**Pull request by solidw**
# Pull Request/Issue Resolution

**Description of Change:**
`ReactAppboy.logPurchase` has different parameter type. I updated it based on [appboy-react-sdk](https://github.com/Appboy/appboy-react-sdk/blob/master/index.d.ts#L200) and [example](https://github.com/Appboy/appboy-react-sdk/blob/master/AppboyProject/AppboyProject.js#L378).

Closes #**nothing**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [x] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [x] Check that all links work!
- [x] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/react_native/android_and_fireos/analytics/revenue_tracking.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 nav_title: Logging Purchases
-platform: React Braze
```
Comment (solidw) [inline _docs/_developer_guide/platform_integration_guides/react_native/android_and_fireos/analytics/revenue_tracking.md:3]: Only this page named React Braze

File: _docs/_developer_guide/platform_integration_guides/react_native/android_and_fireos/analytics/revenue_tracking.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ See [the Android integration instructions][1] for in depth discussion of revenue
 ```javascript
 var properties = {};
 properties["KeyOne"] = "ValueOne";
-ReactAppboy.logPurchase("testPurchaseWithNullCurrency", 10, null, 5, properties);
+ReactAppboy.logPurchase("productId", "10.99", "USD", 5, properties);
```
Comment (solidw) [inline _docs/_developer_guide/platform_integration_guides/react_native/android_and_fireos/analytics/revenue_tracking.md:14]: [logPurchase's parameter type.](https://github.com/Appboy/appboy-react-sdk/blob/master/index.d.ts#L200)
```ts
export function logPurchase(
  productId: string,
  price: string,
  currencyCode: string,
  quantity: number,
  purchaseProperties?: object
): void;
```

**Comment by solidw** [comment]
Minor issue about logPurchase function in React Native document.
@timothy-kim @KellieHawks

**Review by KellieHawks — APPROVED** [review]
Thank you!

**Pull request merged**